run_server.py

```diff
@@ -16,85 +16,33 @@
 import os
 from typing import Sequence
 
-from absl import app
-from absl import flags
-
+import jetstream_pt
+from absl import app, flags
 from jetstream.core import server_lib
 from jetstream.core.config_lib import ServerConfig
+from jetstream_pt.config import (
+    FLAGS,
+    create_engine_from_config_flags,
+    define_common_flags,
+    define_profiling_flags,
+)
 
-import jetstream_pt
-
+define_common_flags()
+define_profiling_flags()
 
-_PORT = flags.DEFINE_integer("port", 9000, "port to listen on")
```

Comment:
we should leave the thread / port flags in this file instead of in the config file.

Reply:
Those 2 are defined in line 33 and 34, used a different way to define it and the global var can be avoided. Accessible by `FLAGS.port`, `FLAGS is an exisiting global from absl.flag